compiler/tests/generated/exports/service_comm.h

```diff
@@ -1162,4 +1162,164 @@ namespace tests
     };
     
 
+    class Bar
```

Comment:
This isn't actually doing anything with the base service is it? Perhaps you could isolate out the tests around the experimental syntax into a separate .bond file that doesn't impact the mainline tests?